t/ui/18-tests-details.t

```diff
@@ -673,5 +674,30 @@ subtest 'archived icon' => sub {
     is $t->tx->res->dom->find('#job-archived-badge')->size, 1, 'archived icon shown if job is archived';
 };
 
+subtest 'test duration' => sub {
+    $start = DateTime->new(
+        year       => 2021,
+        month      => 09,
+        day        => 14,
+        hour       => 15,
+        minute     => 0,
+        second     => 0,
+        nanosecond => 0,
+        time_zone  => 'UTC',
+    );
+    $end = DateTime->new(
+        year       => 2021,
+        month      => 09,
+        day        => 16,
+        hour       => 17,
+        minute     => 30,
+        second     => 0,
+        nanosecond => 0,
+        time_zone  => 'UTC',
+    );
+    my $duration = $t->app->format_time_duration($end - $start);
+    $duration =~ /2 days 02:30 hours/;
```

Comment:
```suggestion
    my $duration = $t->app->format_time_duration($end - $start);
    $duration =~ /2 days 02:30 hours/;
    like $duration, qr/2 days 02:30 hours/, 'duration formatted';
```

And maybe the duration can be instantiated in a more direct and compact way.

Reply:
Applied

Reply:
But the line `$duration =~ /2 days 02:30 hours/;` is still there and seems quite useless. And it is really not possible to instantiate the duration object in a more direct and compact way?

Reply:
Fixed.
I guess we could do it in a more compact way, but I fear to loose readability.